container/secrets.go

```diff
@@ -81,5 +86,8 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	if _, err := os.Stat(overrideDir); err == nil {
```

Comment:
@cgwalters I guess we want this union just for `rhsm` and `rhel7.repo` - does `/etc/pki/entitlement` need this?